frontend/packages/core/src/Input/text-field.tsx

```diff
@@ -44,39 +45,42 @@ const StyledAutocomplete = styled(Autocomplete)({
   },
 });
 
-const StyledTextField = styled(BaseTextField)({
-  "--error-color": "#DB3615",
-  height: "unset",
+const TEXT_FIELD_COLOR_MAP = {
```

Comment:
Lets go ahead and add options for `primary`, `secondary`, `info` and `success` that way they're supported as possible options. 

Right now they're available via typing but do nothing which could be a little odd.